x-pack/plugins/lens/public/metric_visualization/expression.tsx

```diff
@@ -90,7 +148,10 @@ export function MetricChart({
       className="lnsMetricExpression__container"
     >
       <AutoScale key={value}>
-        <div data-test-subj="lns_metric_value" style={{ fontSize: '60pt', fontWeight: 600 }}>
+        <div
+          data-test-subj="lns_metric_value"
+          style={{ fontSize: '60pt', fontWeight: 600, ...color }}
+        >
```

Comment:
Minor nitpick, but would it be possible to 1) apply a `border-radius` of `$euiBorderRadius` and 2) add left and right `padding` of `$euiSize` to this element? Doing so would better match the design.